lib/popover/theme.module.scss

```diff
@@ -21,16 +22,28 @@
     white-space: nowrap;
     max-width: 100px;
     border-radius: 5px;
-    background-color: $secondary-white;
+    background-color: #ffffff;
```

Comment:
the bg color should be same as of `popoverArrow` color or vice versa, so that whole thing looks like a single unit.